frontend/public/components/catalog/catalog-items.jsx

```diff
@@ -125,16 +125,45 @@ const getAvailableFilters = (initialFilters) => {
     },
   };
 
+  filters.capabilityLevel = {
+    BasicInstall: {
+      value: 'Basic Install',
+      label: 'Basic Install',
+      active: false,
+    },
+    SeamlessUpgrades: {
+      value: 'Seamless Upgrades',
+      label: 'Seamless Upgrades',
+      active: false,
+    },
+    FullLifecycle: {
+      value: 'Full Lifecycle',
+      label: 'Full Lifecycle',
+      active: false,
+    },
+    DeepInsights: {
+      value: 'Deep Insights',
+      label: 'Deep Insights',
+      active: false,
+    },
+    AutoPilot: {
+      value: 'Auto Pilot',
+      label: 'Auto Pilot',
+      active: false,
+    },
+  };
+
   return filters;
 };
 
 const filterGroupNameMap = {
   kind: 'Type',
+  capabilityLevel: 'Capability Level'
 };
 
 const filterValueMap = {
-  ClusterServiceClass: 'Service Class',
-  ImageStream: 'Source-to-Image',
+  kind: 'kind',
+  capabilityLevel: 'capabilityLevel'
```

Comment:
any reason why this has been changed? Also, I don't see this prop being used anywhere in tile-view-page

Reply:
I've moved this out....